### PR TITLE
TensorFlow Cloud 0.1.6

### PR DESCRIPTION
**Pull request by SinaChavoshi**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Release 0.1.6
+
+* New Module CloudTuner - Implementation of a library for hyperparameter tuning that is built into the KerasTuner and creates a seamless integration with Cloud AI Platform Optimizer Beta as a backend to get suggestions of hyperparameters and run trials.
```
Comment (ucdmkt) [inline RELEASE.md:3]: s/Module/module/

Comment (ucdmkt) [inline RELEASE.md:3]: mind linking to https://cloud.google.com/ai-platform/optimizer/docs/overview ?

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# Release 0.1.6
+
+* New Module CloudTuner - Implementation of a library for hyperparameter tuning that is built into the KerasTuner and creates a seamless integration with Cloud AI Platform Optimizer Beta as a backend to get suggestions of hyperparameters and run trials. 
+* New application Monitoring - TensorFlow extension that exports its metrics to Stackdriver backend, allowing users to monitor the training and inference jobs in real time.
+* New experimental project Cloud_fit - an experimental module that enables training keras models on Google Cloud AI Platform by serializing the model, and datasets for remote execution.
```
Comment (ucdmkt) [inline RELEASE.md:5]: Capital? or cloud_fit?

Comment (ucdmkt) [inline RELEASE.md:5]: Google Cloud AI Platform? Cloud AI Platform?

Even thought it is implied, it's still good to drop link https://cloud.google.com/ai-platform/training/docs/overview

File: RELEASE.md (hunk 1 above)
Comment (SinaChavoshi) [reply RELEASE.md]: done

File: RELEASE.md (hunk 2 above)
Comment (SinaChavoshi) [reply RELEASE.md]: cloud_fit, updated accordingly.

Comment (SinaChavoshi) [reply RELEASE.md]: Done, also updated to Cloud AI Platform

File: RELEASE.md (hunk 1 above)
Comment (SinaChavoshi) [reply RELEASE.md]: done

File: RELEASE.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# Release 0.1.6
+
+* New module CloudTuner - Implementation of a library for hyperparameter tuning that is built into the KerasTuner and creates a seamless integration with [Cloud AI Platform Optimizer](https://cloud.google.com/ai-platform/optimizer/docs/overview) as a backend to get suggestions of hyperparameters and run trials. 
+* New application Monitoring - TensorFlow extension that exports its metrics to Stackdriver backend, allowing users to monitor the training and inference jobs in real time.
+* New experimental project cloud_fit - an experimental module that enables training keras models on [Cloud AI Platform](https://cloud.google.com/ai-platform/training/docs/overview) by serializing the model, and datasets for remote execution.
```
Comment (ucdmkt) [inline RELEASE.md:5]: Sorry, I should have pointed out... since you say 'Cloud AI Platform Optimizer', you should say 'Cloud AI Platform Training'

Comment (SinaChavoshi) [reply RELEASE.md]: Good catch! thank you :) updated accordingly.

**Comment by pavithrasv** [comment]
Thank you for the PR!

**Pull request merged**